projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Transactions.php

```diff
@@ -162,24 +162,28 @@ public function html( $transaction, $metabox ) {
                 <script type="text/javascript">var zbscrmjs_secToken = '<?php echo esc_js( wp_create_nonce( 'zbscrmjs-ajax-nonce' ) ); ?>';</script>
                
                 <style>
-                        @media all and (max-width:699px){
-                        table.wh-metatab{
-                            min-width:100% !important;
-                        }
-                    }  
+						.zbs-infobox {
+							margin: 0px !important;
+							padding: 0px !important;
+						}
+
+						.zbs-infobox.zbs-live i.zbs-help-ico {
+							font-size: unset;
+						}
```

Comment:
Maybe add a bit of specificity and move to `sass/ZeroBSCRM.admin.transactionedit.scss`?

Reply:
Done.